docs/Concepts/Pruning.md

```diff
@@ -0,0 +1,13 @@
+description: Pruning
+<!--- END of page meta data -->
+
+# Pruning
+
+Use pruning to reduce storage required for the chain. As the number of transactions increases, the world state grows. Pruning removes state tries that aren't required.
+
+!!! Caution
+    If you plan to use [private transactions](Privacy/Privacy-Overview.md), do not use pruning.
+
+A full sync to the Ethereum Mainnet with pruning enabled requires 750GB of disk space. To enable pruning, use the [`--pruning-enabled`](../Reference/CLI/CLI-Syntax.md#pruning-enabled) option.
```

Comment:
Perhaps we should be more specific here, it doesn't require 750GB of space *now*, but rather, 750GB is a recommendation to allow for chain state growth over the course of months. Maybe something like: "We recommend that a node be allocated 750GB of disk space to accommodate Ethereum state growth until approximately May 2020 or 600GB until February 2020."

Reply:
Per our conversation in Slack, I've updated this section to include just the recommendation for 750GB.